__mocks__/frontend-common-components.js

```diff
@@ -27,3 +27,4 @@ DDTable.Tr = () => (<span />);
 DDTable.DROP_MEDIUM = 'medium';
 DDTable.DROP_HIGH = 'high';
 DDTable.DROP_LOW = 'low';
+export const getParams = jest.fn().mockReturnValue({ widgetCode: 'code' });
```

Comment:
This will conflict with other tests that will need a different `getParams` outcome. Please mock this using something like this in the test file

```javascript
jest.mock('frontend-common-components',  () => ({ getParams: ... }))
```